extensions/amp-sidebar/1.0/example/index.html

```diff
@@ -1,53 +1,111 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <script
-      type="module"
-      async
-      src="https://cdn.ampproject.org/bento.mjs"
-    ></script>
-    <script nomodule src="https://cdn.ampproject.org/bento.js"></script>
-    <script
-      type="module"
-      async
-      src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.mjs"
-    ></script>
-    <script
-      nomodule
-      async
-      src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.js"
-    ></script>
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="https://cdn.ampproject.org/v0/bento-sidebar-1.0.css"
-    />
-  </head>
-  <body>
-    <bento-sidebar id="sidebar1" side="right">
-      <ul>
-        <li>Nav item 1</li>
-        <li>Nav item 2</li>
-        <li>Nav item 3</li>
-        <li>Nav item 4</li>
-        <li>Nav item 5</li>
-        <li>Nav item 6</li>
-      </ul>
-    </bento-sidebar>
-
-    <div class="buttons" style="margin-top: 8px">
-      <button id="open-sidebar">Open sidebar</button>
-    </div>
-
-    <script>
-      (async () => {
-        const sidebar = document.querySelector('#sidebar1');
-        await customElements.whenDefined('bento-sidebar');
-        const api = await sidebar.getApi();
-
-        // set up button actions
-        document.querySelector('#open-sidebar').onclick = () => api.open();
-      })();
-    </script>
-  </body>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>bento-sidebar</title>
+
+    <head>
+      <script src="https://cdn.ampproject.org/bento.js"></script>
```

Comment:
Intentionally dropping the module/nomodule pattern here?

Reply:
cf. #36992 where it's now:

```html
<script type="module" src="https://cdn.ampproject.org/bento.mjs" crossorigin="anonymous"></script>
<script nomodule src="https://cdn.ampproject.org/bento.js" crossorigin="anonymous"></script>
<script type="module" src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.mjs" crossorigin="anonymous"></script>
<script nomodule src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.js" crossorigin="anonymous"></script>
<link rel="stylesheet" href="https://cdn.ampproject.org/v0/bento-sidebar-1.0.css" crossorigin="anonymous">
```

Reply:
For my own clarity, does this mean all these examples will use the pattern listed here automatically by some means or the example here should be updated with this syntax?

Reply:
Updated. Examples are not updated automatically, and most hero examples still use the the old pattern. We need a Bento Optimizer...